### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by JamilaHajAhmad**

### CODE REVIEW

File: csv_viewer/first-task.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        table = ConsoleTable.new(rows: rows.to_a)
+        puts table
+    end
+end
+
+csv_path = './csv_files' 
+
+fetcher = Fetcher.new(csv_path)
+if fetcher.files.empty?
+    puts "No CSV files found in the directory."
```
Comment (MaiAbuthraa) [inline csv_viewer/first-task.rb:48]: fix the indentation

```suggestion
  puts "No CSV files found in the directory."
```

File: csv_viewer/first-task.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
+require 'tty-prompt'
+require 'csv'
+require 'console_table'
+
+
```
Comment (MaiAbuthraa) [inline csv_viewer/first-task.rb:5]: Good work, but move all these classes to separated files

File: csv_viewer/first-task.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+csv_path = './csv_files' 
+
+fetcher = Fetcher.new(csv_path)
+if fetcher.files.empty?
+    puts "No CSV files found in the directory."
+    exit
+end
+
+selector = Selector.new(fetcher.files)
```
Comment (MaiAbuthraa) [inline csv_viewer/first-task.rb:52]: It's not good to have free code like that. Everything has to be inside a class.

File: csv_viewer/first-task.rb [hunk 4]
```diff
@@ -0,0 +1,61 @@
+require 'tty-prompt'
+require 'csv'
+require 'console_table'
+
```
Comment (MaiAbuthraa) [inline csv_viewer/first-task.rb:4]: where are the Gemfile and Gemfile.lock?

**Review by MaiAbuthraa — CHANGES_REQUESTED** [review]

**Comment by JamilaHajAhmad** [comment]
Hello Ms. Mai,
All of the requested changes has been addressed in my last commit,
Please check it out,
Thank you

**Pull request merged**